CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Fix : error when add a manufacturing order lacking a production process to a production process.
 - Fix : disable create and edit of saleorderline from sale order invoicing wizard.
 - Fix account reconcilable issue when ventilate a refund invoice.
+- Small fixes in stock move view
```

Comment:
Please, complete the description

Reply:
Fix : On the view -> saleOrder, invoice and purchase.. put readonly when status != draft
Fix : On the view -> saleOrder, invoice and purchase.. disable edition when status != draft